docs/components/checkbox.md

```diff
@@ -131,7 +131,7 @@ Anything passed in to the default slot will replace the label prop text
     </div>
     <div>
       <KCheckbox v-model="slots2">
-        I agree to the <a href="#slots">privacy policy</a>.
+        I agree to the&nbsp;<a href="#slots">privacy policy</a>.
```

Comment:
The actual issue here is that the label is `inline-flex` - should it be?

Reply:
Yes, it was defined in `_labels.scss` styles file then moved to component style block